code/src/main/kotlin/com/expediagroup/sdk/core/constant/provider/LoggingMessageProvider.kt

```diff
@@ -15,13 +15,12 @@
  */
 package com.expediagroup.sdk.core.constant.provider
 
-import io.ktor.http.HttpStatusCode
 
 internal object LoggingMessageProvider {
     fun getTokenExpiresInMessage(expiresIn: Int) = "New token expires in $expiresIn seconds"
 
     fun getResponseUnsuccessfulMessage(
-        httpStatusCode: HttpStatusCode,
+        httpStatusCode: Int,
```

Comment:
Why? It is abstracting away some dependencies but it also makes the value free to accept any number, what is status code 7568 for example? Do we need to build an enum or constant for this?

Reply:
Fixed, created enum class `HttpStatus`.